style.css

```diff
@@ -21,8 +21,18 @@
 body { 
 	font-family: RalewayWF;
 	line-height: 1.6;
-	padding: 2em 4em; 
+    margin-inline: auto;
+    width: clamp(16rem, 95vw, 85rem);
+    padding-inline: clamp(1.4rem, 1.5rem, 2rem);
 	}
+
+@media (min-width: 600px) {
+    body {
+        margin: 0 3em 0 3em;
```

Comment:
Until this format is fixed by CSS and deployed, a better i18n solution would be:
margin-inline: 3rem;
margin-block: 0;

Similar for the padding setting, too.

Reply:
Fixed.